docs/concepts/developer-experience.md

```diff
@@ -1,5 +1,5 @@
 # Developer Experience
 
 To create a great application, we need great development experience.
-Fortunately, we can write applications for Cloudflare Workers, Deno, and Bun in TypeScript without having the transpiling to JavaScript.
-Hono is written in TypeScript and can make applications Type-safe.
+Fortunately, we can write applications for Cloudflare Workers, Deno, and Bun in TypeScript without having the need to transpile it to JavaScript.
+Hono is written in TypeScript and can make applications type-safe.
```

Comment:
`Type-Safety` is used in the following:

https://github.com/honojs/website/blob/7489b77a4b9df6614f070537495215585575e166/docs/concepts/stacks.md?plain=1#L10

Please also change this word not to use uppercase.

Reply:
Done. I also found a typo in there and made slight improvements to the docs on Stacks as well. LMK if the changes are to your liking.

Reply:
Thanks!